packages/react/src/hooks/usePagination.ts

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+
+/**
+ * The `usePagination` hook implements simple pagination logic for use with arrays.
+ * @example
+ * ```tsx
+ * const { firstItemIndex, lastItemIndex } = usePagination(3, 9);
+ * const itemsOnPage = items.slice(firsItemIndex, lastItemIndex)
+ * ```
+ * @internal
+ */
+export function usePagination(itemPerPage: number, totalItemCount: number) {
+  const [page, setPage] = useState(1);
```

Comment:
wondering if we should rename this to `currentPage` and return all other pages as well? 🤔

Reply:
`currentPage` is fine by my 👍 
What do you mean by "return all other pages"? At the moment only the index of the page is returned.

Reply:
ah, you're right. I assumed it would return the relevant trackBundles directly. But index is fine by me!